bin/yaml/tools.yaml

```diff
@@ -9,13 +9,28 @@ tools:
     check_exe: bin/cmake --help
     targets:
       - 3.26.1
-      - 3.26.4
-      - 3.27.9
+      - name: 3.26.4
```

Comment:
Please stick to the format of the others. The `symlink` is actively harmful, and is only used as noted below (to distinguish the one `cmake` we use on the site itself to compile things)